dotcom-rendering/cypress/support/commands.js

```diff
@@ -28,19 +28,29 @@ import 'cypress-wait-until';
 
 Cypress.Commands.add('hydrate', () => {
 	return cy
-		.get('gu-island[deferUntil=visible]')
+		.get('gu-island')
 		.each((el) => {
-			cy.wrap(el)
-				.log(`Scrolling to ${el.attr('name')}`)
-				.scrollIntoView({ duration: 100, timeout: 10000 })
-				.should('have.attr', 'data-gu-ready', 'true', {
-					timeout: 30000,
-				});
+			const deferuntil = el.attr('deferuntil');
+			const name = el.attr('name');
+			const defer = el.attr('deferuntil');
+			const islandMeta = `island: ${name} defer: ${defer}`;
+			if (['idle', 'visible', undefined].includes(deferuntil)) {
+				cy.log(`Scrolling to ${islandMeta}`);
+				cy.wrap(el)
+					.scrollIntoView({ duration: 1000, timeout: 30000 })
+					.should('have.attr', 'data-gu-ready', 'true', {
+						timeout: 30000,
```

Comment:
Small readability improvement:

```suggestion
					.scrollIntoView({ duration: 1000, timeout: 30_000 })
					.should('have.attr', 'data-gu-ready', 'true', {
						timeout: 30_000,
```

Reply:
This causes the lint error `Parsing error: Identifier directly after number` so I have removed for now.